Blockzilla/BrowserViewController.swift

```diff
@@ -822,6 +822,15 @@ extension BrowserViewController: URLBarDelegate {
         default: break
         }
     }
+    
+    func urlBarDidSingleTap(_: URLBar) {
+        guard !urlBar.isEditing else { return }
+        
+        switch scrollBarState {
+        case .collapsed: showToolbars()
+        default: break
+        }
+    }
 
```

Comment:
Why is this version so much more simpler than the current implementation of `urlBarDidPressScrollTop()` ?

```
    func urlBarDidPressScrollTop(_: URLBar, tap: UITapGestureRecognizer) {
        guard !urlBar.isEditing else { return }

        switch scrollBarState {
        case .expanded:
            let y = tap.location(in: urlBar).y

            // If the tap is greater than this threshold, the user wants to type in the URL bar
            if y >= 10 {
                urlBar.activateTextField()
                return
            }

            // Just scroll the vertical position so the page doesn't appear under
            // the notch on the iPhone X
            var point = webViewController.scrollView.contentOffset
            point.y = 0
            webViewController.scrollView.setContentOffset(point, animated: true)
        case .collapsed: showToolbars()
        default: break
        }
    }
```

Do we not care about the `.collapsed` state anymore?